README.md

```diff
@@ -51,6 +51,13 @@ npm install string-ts
   - [toSnakeCase](#tosnakecase)
   - [toConstantCase](#toconstantcase)
   - [toTitleCase](#totitlecase)
+- [Strongly-typed deep transformation of objects](#strongly-typed-transformation-of-objects)
```

Comment:
```suggestion
- [Strongly-typed shallow transformation of objects](#strongly-typed-transformation-of-objects)
```

Reply:
done in [6cde47a](https://github.com/gustavoguichard/string-ts/pull/16/commits/6cde47a1a68c50d506dc516c25ad05ce56454ed6)